pkg/controller/certificates/issuing/issuing_controller.go

```diff
@@ -390,9 +390,13 @@ func (c *controller) issueCertificate(ctx context.Context, nextRevision int, crt
 		return err
 	}
 	secretData := internal.SecretData{
-		PrivateKey:  pkData,
-		Certificate: req.Status.Certificate,
-		CA:          req.Status.CA,
+		PrivateKey:      pkData,
+		Certificate:     req.Status.Certificate,
+		CA:              req.Status.CA,
+		CertificateName: crt.Name,
+		IssuerName:      req.Spec.IssuerRef.Name,
+		IssuerKind:      req.Spec.IssuerRef.Kind,
+		IssuerGroup:     req.Spec.IssuerRef.Group,
```

Comment:
The IssuerName, IssuerKind, IssuerGroup would previously have been copied from the Certificate.
Now they are copied from the CertificateRequest.
Why? And does it matter?

Reply:
I think it is safer to copy from the request. Those fields are immutable.